### PR TITLE
mgr/cephadm: Fixed warn_on_stray_hosts setting

### PR DESCRIPTION
**Pull request by ahaase-c4u**
…set to false.

Fixes: https://tracker.ceph.com/issues/45065

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
DCO is missing. Mind doing a 

```
git checkout 45065
git commit --amend -s
git push -f
``?
?

File: src/pybind/mgr/cephadm/module.py [hunk 1]
```diff
@@ -1069,23 +1069,25 @@ def _check_for_strays(self):
...
-            if daemon_detail:
-                self.health_checks['CEPHADM_STRAY_DAEMON'] = {
-                    'severity': 'warning',
-                    'summary': '%d stray daemons(s) not managed by cephadm' % (
-                        len(daemon_detail)),
-                    'count': len(daemon_detail),
-                    'detail': daemon_detail,
-                }
+            if self.warn_on_stray_hosts:
+                if host_detail:
```
Comment (sebastian-philipp) [inline src/pybind/mgr/cephadm/module.py:1073]:
```suggestion
            if self.warn_on_stray_hosts and host_detail:
```

File: src/pybind/mgr/cephadm/module.py [hunk 2]
```diff
@@ -1069,23 +1069,25 @@ def _check_for_strays(self):
...
+                    self.health_checks['CEPHADM_STRAY_HOST'] = {
+                        'severity': 'warning',
+                        'summary': '%d stray host(s) with %s daemon(s) '
+                        'not managed by cephadm' % (
+                            len(host_detail), host_num_daemons),
+                        'count': len(host_detail),
+                        'detail': host_detail,
+                    }
+            if self.warn_on_stray_daemons:
+                if daemon_detail:
```
Comment (sebastian-philipp) [inline src/pybind/mgr/cephadm/module.py:1083]:
```suggestion
            if self.warn_on_stray_daemons and daemon_detail:
```

**Comment by mgfritch** [comment]
QA run successful. failure is unrelated.

http://pulpito.ceph.com/mgfritch-2020-04-15_00:22:19-rados-wip-mgfritch-testing-2020-04-14-1709-distro-basic-smithi/

**Pull request merged**